### ISSUE TITLE
Pressing the 'stop

### ISSUE THREAD

**Issue by BarbourSmith** [MaslowCNC/GroundControl#577]

**Comment by blurfl** [comment]
If the appropriate fix is changing the text of the button to 'Emergancy\r\nStop', I can take care of it if you want...

**Comment by BarbourSmith** [comment]
I want to have a go at trying to find out why it's giving the wrong measurement, I'd like using the stop button to be an option without restarting the calibration process, but if that doesn't pan out let's go with the emergency stop option

**Comment by krkeegan** [comment]
Agree, that is odd. is the B10 measure command reading the target and not the current position?

**Comment by BarbourSmith** [comment]
That's a good suggestion, that would explain it

**Comment by blurfl** [comment]
May be related - I notic ed yesterday that after pressing 'Stop' during a cut run, If I used 'GoTo' to begin the cut at the step before the interrupteion, the Z-axis wouldn't respond to gcode commands (giggled but didn't move). Didn't spend enough time on it to open an issue on it though...

**Comment by krkeegan** [comment]
> the Z-axis wouldn't respond to gcode commands

That is weird, I don't have a good explanation for why this would happen.

**Comment by blurfl** [comment]
I'll go back and try to come up with enough for a separate issue when time permits.
